app/src/main/java/com/infomaniak/mail/utils/LogoutUser.kt

```diff
@@ -56,6 +56,8 @@ class LogoutUser @Inject constructor(
         localSettings.removeRegisteredFirebaseUser(userId = user.id)
 
         if (user.id == AccountUtils.currentUserId) {
+            AccountUtils.currentUser = null // Prevent that the user current no longer exists
```

Comment:
I'm not sure what you meant with this comment. Maybe you meant `Prevent that the current user no longer exists` but still, if you meant that sentence, the user is set to `null` which means the user is no longer. So what is the actual meaning of this comment?

Reply:
```suggestion
            AccountUtils.currentUser = null // Inform that there is no more current user
```